CONTRIBUTING.md

```diff
@@ -94,15 +92,12 @@ or run manually:
 
 ```bash
 docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons gcr.io/tensorflow-testing/nosla-cuda10.1-cudnn7-ubuntu16.04-manylinux2010 /bin/bash
-export TF_NEED_CUDA=1
-./configure.sh  # Links project with TensorFlow dependency
 
-bazel test -c opt -k \
---test_timeout 300,450,1200,3600 \
---crosstool_top=//build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda10.1:toolchain \
---test_output=all \
---jobs=1 \
-//tensorflow_addons/...
+export TF_GPU_COUNT=4 # Specify number of GPUs available
+export TF_TESTS_PER_GPU=8 # Specify number of tests per GPU
+export TF_PER_DEVICE_MEMORY_LIMIT_MB=1024 # Limit the memory used per test
+
+bash tools/ci_testing/addons_gpu.sh
```

Comment:
Thinking a bit more on this, I think it's unlikely that developers will be using multi-gpus for testing. It's even less useful because you only need to run one target for most changes. 

How do you feel about making the `addons_gpu.sh` default to single GPU if some `kokoro` flag is not set. Then we can just remove this export information (or document it in the script) so that it's less confusing for those who just want to test a single GPU target change.

Reply:
So I tested an if these ENV variables are not set (but only one GPU exists) then the test still runs successfully with `addons_gpu.sh`.